scripts/windows/VERIFICATION.txt

```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+To verify the logdna-agent.exe:
+ 
+1. Download <https://s3.amazonaws.com/repo.logdna.com/win/logdna-agent.latest.zip> which includes "logdna-agent.exe"
+2. You can use one of the following methods to obtain the checksum
+  - Use powershell function 'Get-Filehash'
+  - Use chocolatey utility 'checksum.exe'
+ 
+  checksum type: sha256
+  checksum: 9daf22c15f4f42e8c7ac4cdf73c26f7154d325d0defe14ece18d1203a9c7afd1
+ 
+This package is published by the LogDNA. You can build a binary directly
+by following instructions here:
```

Comment:
Here you need to *insert* a "the", so "by following the instructions here:"

Reply:
Done!